htdocs/change.php

```diff
@@ -158,8 +158,8 @@
 #==============================================================================
 # Check password strength
 #==============================================================================
-if ( !$result ) {
-    $result = check_password_strength( $newpassword, $oldpassword, $pwd_policy_config, $login, $entry_array );
+if ( $result === "" ) {
```

Comment:
this has to be `if ( !$result ) {`.

Besides that, everything seams to be working as expected.

Reply:
Perfect, I have just fixed it in last commit. I am going to merge this